src/smart-auth/index.ts

```diff
@@ -209,6 +229,10 @@ export const getAccessTokenFromClientCredentialFlow = async (
   smartAuthProvider: SmartAuthProvider,
   scope?: string[]
```

Comment:
Ah I see - you can ignore my comment above if the plan is to always pass in scope as a param 👍

Reply:
It will pass whatever value you use in this argument _or_ in `auth.tokenParams` of the SmartAuthProvider but otherwise will send nothing